src/ts/component/menu/dataview/calendar.tsx

```diff
@@ -94,11 +94,27 @@ const MenuCalendar = observer(class MenuCalendar extends React.Component<I.Menu>
 						return (
 							<div 
 								key={i} 
+								id={`day-${item.y}-${item.m}-${item.d}`}
 								className={cn.join(' ')} 
 								onClick={(e: any) => { 
 									e.stopPropagation();
 									this.setValue(U.Date.timestamp(item.y, item.m, item.d), true, true); 
 								}}
+								onContextMenu={(e: any) => {
```

Comment:
- Move to separate handler
- Add `#{getId()}` to selector
- Remove width
- Format options array, add icon 'expand', change name to Open as Object
- Change method to 'auto'